covariance/covarianceXsec.h

```diff
@@ -12,42 +12,52 @@
 // MaCh3 includes
 #include "covarianceBase.h"
 #include "samplePDF/Structs.h"
-#include "throwParms/ThrowParms.h"
+
+#include "yaml-cpp/yaml.h"
 
 class covarianceXsec : public covarianceBase {
 
   public:
-  covarianceXsec(const char *name, const char *file, double threshold=-1,int firstpcapar=-999,int lastpcapar=-999);
+  covarianceXsec(const char *name, const char *file, double threshold=-1,int FirstPCAdpar=-999,int LastPCAdpar=-999);
+  covarianceXsec(const char *YAMLFile);
```

Comment:
Why there is no Yaml in PCA consturctor?

Reply:
I haven't got round to validation the PCA with the YAML-ified systematics. This should be pretty easy but for now I have skipped it.